adb/common_cli.py

```diff
@@ -142,17 +141,16 @@ def _RunMethod(dev, args, extra):
   return 0
 
 
-def StartCli(args, device_factory, extra=None, **device_kwargs):
+def StartCli(args, adb_commands, extra=None, **device_kwargs):
   """Starts a common CLI interface for this usb path and protocol."""
   try:
-    dev = device_factory(
-        port_path=args.port_path, serial=args.serial,
-        default_timeout_ms=args.timeout_ms, **device_kwargs)
+    dev = adb_commands()
+    dev.ConnectDevice(port_path=args.port_path, serial=args.serial, default_timeout_ms=args.timeout_ms, **device_kwargs)
   except usb_exceptions.DeviceNotFoundError as e:
-    print >> sys.stderr, 'No device found: %s' % e
+    print('No device found: {}'.format(e), file=sys.stderr)
```

Comment:
`from __future__ import print_function`
Did you test any of this on Python 2?

Reply:
Fixed